content/en/agent/guide/how_remote_config_works.md

```diff
@@ -75,6 +75,11 @@ The following products and features are supported with Remote Config:
 
 - **Automatic default agent rule updates**: Automatically receive and update the default Agent rules maintained by Datadog as new Agent detections and enhancements are released. 
 
+### Observability Pipelines (OP)
+<div class="alert alert-info">This feature is in private beta.</div>
+
+- **Remotely update and deploy [Observability Pipelines Worker (OPW)][10]**: Build and update pipelines in the UI, remotely rolling out the new configuration to OPW running in your environment.
```

Comment:
Let's change to:

**Remotely deploy and update [Observability Pipelines Workers (OPW)][10]**: Build and edit pipelines in the Datadog UI, rolling out your configuration changes to OPW instances running in your environment.